{{cookiecutter.project_slug}}/tox.ini

```diff
@@ -1,30 +1,41 @@
 [tox]
-envlist = py35, py36, py37, py38, flake8
+envlist = py{36,37,38}, black{%- if cookiecutter.make_docs ==  'y' %}, docs{%- endif %}, py38-{macOS,Windows}
 
 [travis]
 python =
-    3.8: py38
-    3.7: py37
-    3.6: py36
-    3.5: py35
+    3.6: black
+    {% if cookiecutter.make_docs ==  'y' -%}
+    3.6: docs
+    {%- endif %}
 
-[testenv:flake8]
-basepython = python
-deps = flake8
-commands = flake8 {{ cookiecutter.project_slug }} tests
+[testenv:black]
+skip_install = True
+deps =
+    flake8
+    black
+commands =
+    flake8 {{ cookiecutter.project_slug }} tests
```

Comment:
why not `make lint` in there to avoid repeating again the same commands in the Makefile?

Reply:
Oh dang. Not a bad idea!